README.md

```diff
@@ -30,25 +54,225 @@ npm install --save ipfsd-ctl
 
 IPFS daemons are already easy to start and stop, but this module is here to do it from JavaScript itself.
 
+### Local node
+
 ```js
 // Start a disposable node, and get access to the api
-// print the node id, and kill the temporary daemon
+// print the node id, and stop the temporary daemon
 
 // IPFS_PATH will point to /tmp/ipfs_***** and will be
 // cleaned up when the process exits.
 
-var ipfsd = require('ipfsd-ctl')
+const daemonFactory = require('ipfsd-ctl')
+const local = daemonFactory.localController
 
-ipfsd.disposableApi(function (err, ipfs) {
-  ipfs.id(function (err, id) {
+local.spawn(function (err, ipfsd) {
+  const ipfsCtl = ipfsd.ctl
+  const ipfsCtrl = ipfsd.ctrl
+  ipfsCtl.id(function (err, id) {
     console.log(id)
-    process.exit()
+    ipfsCtrl.stopDaemon()
```

Comment:
Is there any call for cleanUp??

Reply:
yes, there is `ipfsCtrl.shutdown` which will clean up the tmp repo created for the disposable node.

Reply:
What is the difference between:

- .stopDaemon
- .shutdown

If I stopDaemon, can I then later clean up?

Reply:
`stopDaemon` will `kill` the process which would call `shutdown`, that is registered as a process exit handle if the daemon was started as a disposable node. If the node is not disposable, shutdown will not be called and the repo not destroyed (what we want).

Reply:
I think `shutdown` would be better if renamed to `cleanup`.

Reply:
> If I stopDaemon, can I then later clean up?

Yep, that just simply calls `rimraf` on the repo, so calling after the daemon is stopped is supposed to work.